free-programming-books-tr.md

```diff
@@ -26,18 +26,22 @@
 ### Android
 
 * [Android Geleceği Yazanlar](https://gelecegiyazanlar.turkcell.com.tr/konu/android)
+* [Android Dersleri] (http://umiitkose.com/android/)
```

Comment:
Hello, the reason your Travis CI build fails is because of improper ordering and spacing

```suggestion
* [Android Dersleri] (http://umiitkose.com/android/)
* [Android Geleceği Yazanlar](https://gelecegiyazanlar.turkcell.com.tr/konu/android)
```

Swap the ordering so it is alphabetical order as suggested by the [CONTRIBUTING](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md) file.